src/euphorie/content/tests/test_utils.py

```diff
@@ -92,8 +92,12 @@ def testMixed(self):
 
     def testUgly(self):
         self.assertEqual(
-            parse_scaled_answers("\n\n hello world |  7\n\n\n  b  \n\n\n"),
-            [{"text": "hello world", "value": "7"}, {"text": "b", "value": "2"}],
+            parse_scaled_answers("\n\n hello world |  7\n\n\n  b  \n\n\nc|"),
```

Comment:
Good new test case.
But this loses the check for what happens with one or more empty lines at the end.

```suggestion
            parse_scaled_answers("\n\n hello world |  7\n\n\n  b  \n\n\nc|\n\n\n"),
```